scripts-available/CDB_CartodbfyTable.sql

```diff
@@ -1070,15 +1099,11 @@ BEGIN
       )
       SELECT ', ST_Transform(' 
             || t.missing_srid_start || t.attname || t.missing_srid_end
-            || ',4326)::Geometry(' 
-            || t.geomtype 
-            || ',4326) AS '
+            || ',4326)::Geometry(GEOMETRY,4326) AS '
             || const.geomcol
```

Comment:
There's no real downside to having geometry unrestricted by type, since the only renderer hitting the tables in cartodb is one that we expect to be able to deal with multi-type inputs; so yeah, sorry for the noise, just being a big baby.